flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/LookupJoinITCase.scala

```diff
@@ -84,6 +84,20 @@ class LookupJoinITCase extends StreamingTestBase {
     .field("name", Types.STRING)
     .build()
 
+  val userDataWithSameAge = List(
+    (11, 1L, "Julian"),
+    (22, 2L, "Jark"),
+    (33, 3L, "Fabian"),
+    (11, 4L, "Hello world")
+  )
+
+  val userWithSameAgeTableSourceWith2Keys = InMemoryLookupableTableSource.builder()
```

Comment:
Could we reuse the existing testing data or improve the existing data? 

We should avoid introducing to much testing data.

Reply:
Good Idea. I improved the existing data.